source/_posts/2017-03-15-what-the-sql-recursive.markdown

```diff
@@ -0,0 +1,192 @@
+---
+layout: post
+title: "What the SQL?!? Recursive"
+date: 2017-03-15 12:30:00-0600
+comments: true
+categories: talk sql
+published: true
+---
+
+{% img featured /images/i_heart_recursion.png 1804 1195 'What the SQL?!? Recursive' %}
+
+# What the SQL?!? Recursive
+
+Today's "What the SQL?!?" features the keyword `RECURSIVE`. This clause allows
+us to elegantly select results from the previous results from previous results from
+previous results...
```

Comment:
Should be: `This clause allows us to elegantly select results from the previous results from THE previous results from THE...`